go.mod

```diff
@@ -1,262 +1,267 @@
 module github.com/observiq/bindplane-agent
 
-go 1.20
+go 1.21
```

Comment:
Are we updating the go version on purpose here? totally fine if so, just noticed normally we keep it at 1.20

Reply:
Yes. Upstream updated go, so we don't have a choice 😅

Reply:
Fair enough